client/question/domain/FeedbackObjectFactory.js

```diff
@@ -129,13 +129,25 @@ tie.factory('FeedbackObjectFactory', [
 
     /**
      * Appends a FeedbackParagraph of type SyntaxError to the _paragraphs
-     * Array
+     * Array.
      *
-     * @param {string} text String describing the syntax error
+     * @param {string} outputString String describing the syntax error.
```

Comment:
I think you meant to rename the method below, not this one...

Reply:
Ah. Sorry. Fixed.